drivers/dma/Kconfig.stm32

```diff
@@ -23,10 +23,17 @@ config DMA_STM32_V1
 
 config DMA_STM32_V2
 	bool
-	depends on SOC_SERIES_STM32F0X || SOC_SERIES_STM32F1X || SOC_SERIES_STM32F3X || SOC_SERIES_STM32L0X || SOC_SERIES_STM32L4X ||SOC_SERIES_STM32WBX
```

Comment:
Miss a ">" at the end of the "Signed-off-by" for this commit.
Signed-off-by: Francois Ramu <francois.ramu@st.com